entrypoint.sh

```diff
@@ -42,7 +42,7 @@ if [ "$GMAIL_USER" -a "$GMAIL_PASSWORD" ]; then
 elif [ "$SES_USER" -a "$SES_PASSWORD" ]; then
 	opts+=(
 		dc_eximconfig_configtype 'smarthost'
-		dc_smarthost "email-smtp.${SES_REGION:=us-east-1}.amazonaws.com::587"
+		dc_smarthost "email-smtp.${SES_REGION:=us-east-1}.amazonaws.com::${SES_PORT=587}"
```

Comment:
@psychopenguin thanks for the PR, can we do `${SES_PORT:=587}` to keep it consistent with previous variable subsitution?